src/Aspire.Hosting/Dcp/DcpOptions.cs

```diff
@@ -31,6 +32,8 @@ internal sealed class DcpOptions
     /// </example>
     public string? ExtensionsPath { get; set; }
 
+    public string? DashboardPath { get; set; }
```

Comment:
Would be great to see API doc on this member, similar to that of the prior two properties, with an example value.

Reply:
Addressed in #1777